apps/discordbot/bot.py

```diff
@@ -37,16 +42,33 @@ async def on_ready(self):
             self.test_channel = get(self.test_guild.channels, id=TEST_CHANNEL_ID)
             self.hoser_role = get(self.csua_guild.roles, id=HOSER_ROLE_ID)
 
+    async def verify_member_email(self, user): 
```

Comment:
Where is this being called?

Reply:
It looks like you aren't calling this anywhere ... ?

Reply:
Not sure how to test it properly

Reply:
fixed and tested